lib/trento/application/event_handlers/roll_up_event_handler.ex

```diff
@@ -13,9 +13,23 @@ defmodule Trento.RollUpEventHandler do
 
   alias Trento.Domain.Events.{
     ClusterRolledUp,
-    ClusterRollUpRequested
+    ClusterRollUpRequested,
+    HostRolledUp,
+    HostRollUpRequested
   }
 
+  def handle(%HostRollUpRequested{host_id: stream_id, snapshot: snapshot}, _) do
```

Comment:
Minor thing: we could put this `handle` after the cluster handle function to keep alphabetical consistency